boost.spec

```diff
@@ -1,83 +1,86 @@
-### RPM external boost 1.51.0
-%define boostver _%(echo %realversion | tr . _)
-Source: http://switch.dl.sourceforge.net/project/%{n}/%{n}/%{v}/%{n}%{boostver}.tar.gz
-%define online %(case %cmsplatf in (*onl_*_*) echo true;; (*) echo false;; esac)
+### RPM external boost 1.57.0
+
+%define tag 040a06a
+%define branch cms/v%realversion
+%define github_user cms-externals
+Source: git+https://github.com/%github_user/%n.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}.tgz
 
 %if "%{?cms_cxxflags:set}" != "set"
-%define cms_cxxflags -std=c++0x -O2
+%define cms_cxxflags -std=c++11 -O2
 %endif
 
-Requires: python bz2lib
-%if "%online" != "true"
-Requires: zlib
-%endif
-Patch0: boost-1.47.0-fix-strict-overflow
-Patch1: boost-1.47.0-fix-unused
-Patch2: boost-1.49.0-explicit_stored_group
+Requires: python bz2lib zlib
 
 %prep
-%setup -n %{n}%{boostver}
-%patch0 -p1
-%patch1 -p1
-%patch2 -p1
+%setup -n %{n}
```

Comment:
I was getting the error: 
<pre> cd: boost: No such file or directory </pre>

This should be changed to %setup -n %{n}-%{realversion}